docs/wiki/_Sidebar.md

```diff
@@ -12,6 +12,7 @@
 - [Pipelines](./Pipelines)
   - [Design](./PipelinesDesign)
     - [Parameter File Tokens](./ParameterFileTokens)
+    - [Removal action](./PipelineRemovalAction)
```

Comment:
I'd suggest to remove both these items (Parameter File Tokens + Removal action) from the sidebar to keep that one as easy as possible. The full picture in all its glory should however be visible on the `home.md` page. 
Any thoughts?

Reply:
Discussed offline. We need @eriqua for this too 😄 
For the moment, I will add the structure both in home and  _sidebar

Reply:
I agree on the idea of brainstorming a wiki restructure outside of this PR. I'm sure we can provide a more holistic view. For instance I'd merge testing and pipeline sections as in the end we provide 2 main things: 
1. module library (modules section)
2. built-in testing pipelines (everything else we have in the wiki)

but let's discuss that with the whole team in a dedicated session.

To the matter at hand I'd strongly avoid to update the sidebar manually, this can easily diverge. 
There are multiple places which we'd need to update every time we add a new page: the home, the sidebar but also the parent page of the new section. e.g. in this case also the `./Pipelines` page should be aligned.

For this reason I'd suggest to either limit the sidebar to main sections or to find a way (outside of this PR) to update it automatically based on the main table of contents in the home section, leveraging the platform.wiki-sync pipeline.

Reply:
The main reason I disagree with not having the full structure in the sidebar is that, in my opinion, you need a straightforward way to see where you are in the documentation structure and to navigate somewhere else.
Ideally this would need to resemble the official documentation. If we can't easily get the same, I would try to have something as similar as possible.
I would suggest not to focus on the **what** but the **how** a user can use it. Technical details could be in the section dedicated to the "contributor" or a completely separate one
I agree on the need to update it automatically and/or only in one place.
![image](https://user-images.githubusercontent.com/22790241/147228803-0ef8ac34-7fee-4ba3-9639-3c8aaee6423f.png)

Reply:
Question is - should this go into the release today or not. I guess not ;)

Reply:
Resolving as discussion can continue in issue #836